app/models/concerns/sdg/relatable.rb

```diff
@@ -75,18 +75,24 @@ def sdg_target_list
 
   def sdg_related_list
     sdg_goals.order(:code).map do |goal|
-      [goal, sdg_global_targets.where(goal: goal).sort]
+      global_targets = sdg_global_targets.where(goal: goal)
+      local_targets = sdg_local_targets.where(goal: goal)
+      targets_related_with_goal = global_targets + local_targets
+      [goal, targets_related_with_goal.sort]
     end.flatten.map(&:code).join(", ")
```

Comment:
Maybe we could simplify this part a bit :thinking:.

```ruby
  def sdg_related_list
    related_sdgs.sort.map(&:code).join(", ")
  end
```
Feel free to rename `related_sdgs` so the naming is consistent.

Here's a possible implementation:

```diff
diff --git a/app/models/sdg/goal.rb b/app/models/sdg/goal.rb
index 544046af0e..85f65ebb47 100644
--- a/app/models/sdg/goal.rb
+++ b/app/models/sdg/goal.rb
@@ -18,6 +18,14 @@ class SDG::Goal < ApplicationRecord
     I18n.t("sdg.goals.goal_#{code}.description")
   end
 
+  def <=>(goal_or_target)
+    if goal_or_target.class == self.class
+      code <=> goal_or_target.code
+    elsif goal_or_target.respond_to?(:goal)
+      [self, -1] <=> [goal_or_target.goal, 1]
+    end
+  end
+
   def self.[](code)
     find_by!(code: code)
   end
diff --git a/app/models/sdg/local_target.rb b/app/models/sdg/local_target.rb
index fbc9cbce7b..b0ce0896a9 100644
--- a/app/models/sdg/local_target.rb
+++ b/app/models/sdg/local_target.rb
@@ -21,11 +21,11 @@ class SDG::LocalTarget < ApplicationRecord
     find_by!(code: code)
   end
 
-  def <=>(any_target)
-    if any_target.class == self.class
-      [target, numeric_subcode] <=> [any_target.target, any_target.numeric_subcode]
-    elsif any_target.class == target.class
-      -1 * (any_target <=> self)
+  def <=>(goal_or_target)
+    if goal_or_target.class == self.class
+      [target, numeric_subcode] <=> [goal_or_target.target, goal_or_target.numeric_subcode]
+    elsif [target.class, goal.class].include?(goal_or_target.class)
+      -1 * (goal_or_target <=> self)
     end
   end
 
diff --git a/app/models/sdg/target.rb b/app/models/sdg/target.rb
index ee1e96a45f..e20536e1b9 100644
--- a/app/models/sdg/target.rb
+++ b/app/models/sdg/target.rb
@@ -12,11 +12,13 @@ class SDG::Target < ApplicationRecord
     I18n.t("sdg.goals.goal_#{goal.code}.targets.target_#{code_key}.title")
   end
 
-  def <=>(any_target)
-    if any_target.class == self.class
-      [goal.code, numeric_subcode] <=> [any_target.goal.code, any_target.numeric_subcode]
-    elsif any_target.class.name == "SDG::LocalTarget"
-      [self, -1] <=> [any_target.target, 1]
+  def <=>(goal_or_target)
+    if goal_or_target.class == self.class
+      [goal.code, numeric_subcode] <=> [goal_or_target.goal.code, goal_or_target.numeric_subcode]
+    elsif goal_or_target.class == goal.class
+      -1 * (goal_or_target <=> self)
+    elsif goal_or_target.class.name == "SDG::LocalTarget"
+      [self, -1] <=> [goal_or_target.target, 1]
     end
   end
```